doc/paper/paper.md

```diff
@@ -16,13 +16,13 @@ authors:
     affiliation: "3, 4"
   - name: Filip Waselewski
     orcid: 0000-0003-0729-5879
-    affiliation:
+    affiliation: 6
   - name: Kai Wohlfahrt
     orcid: 0000-0002-0970-5539
     affiliation: 5
-  - name: Aaron O'Leary
+  - name: Aaron O&#8217;Leary
```

Comment:
The apostrophe was causing issues in the paper compilation. Using the HTML character here to fix this.